src/converters.jl

```diff
@@ -24,7 +24,7 @@ A converter that runs the SBML conversion routine, with specified conversion
 options. The argument is a vector of pairs to allow specifying the order of
 conversions.
 """
-libsbml_convert(conversion_options::Vector{Pair{String,Dict{String,String}}}) =
+libsbml_convert(conversion_options::Vector{Pair{String,Dict{Any,Any}}}) =  # @MK: [(string(k), string(v), k, v) for (k, v) in Dict()] yields NTuple{4, Any}[]
```

Comment:
why this? The code relies on the fact that the content of the dictionary is strings.

Reply:
(line 34,35 below is going to fail on any other type)

Reply:
@exaexa : I did this because e.g. libsbml_convert("promoteLocalParameters") fails with a method error (complaining that there is not method `libsbml_convert(conversion_options::Vector{Pair{String,Dict{Any,Any}}})` if I remember correctly). This is because `libsbml_convert([converter => Dict(string(k) => string(v) for (k, v) in kwargs)])` produces a `Dict{Any,Any}` `kwargs` is an empty Dict.

Reply:
ah okay, then we need to type it better on the call site I'd say. I'll patch the call.

```julia
Dict{String,String}(string(k) => string(v) ............. )
```